src/mono/wasm/Makefile

```diff
@@ -59,6 +61,10 @@ MONO_LIBS = \
 	$(ICU_LIBDIR)/libicudata.a
 
 EMCC_FLAGS=--profiling-funcs -s WASM=1 -s ALLOW_MEMORY_GROWTH=1 -s BINARYEN=1 -s ALIASING_FUNCTION_POINTERS=0 -s NO_EXIT_RUNTIME=1 -s "EXTRA_EXPORTED_RUNTIME_METHODS=['ccall', 'FS_createPath', 'FS_createDataFile', 'cwrap', 'setValue', 'getValue', 'UTF8ToString', 'UTF8ArrayToString', 'addFunction']" -s "EXPORTED_FUNCTIONS=['_putchar']" --source-map-base http://example.com  -emit-llvm -s FORCE_FILESYSTEM=1 -s USE_ZLIB=1
+ifeq ($(ENABLE_ES6_UPPER),TRUE)
+    EMCC_FLAGS+=-s MODULARIZE=1 -s EXPORT_ES6
```

Comment:
Does this need a leading space to avoid turning the trailing ```-s USE_ZLIB=1``` from the previous line into ```-s USE_ZLIB=1-s MODULARIZE=1```, which I assume won't parse correctly?

Reply:
`+=` operator in GNU make adds a single space in between, if the variable had a value already. (also `emcc(1)` fails if those two values are joined)